hazelcast/src/main/java/com/hazelcast/client/impl/ClientEndpointImpl.java

```diff
@@ -282,6 +282,18 @@ public String toString() {
                 + '}';
     }
 
+    @Override
+    public String toSecureString() {
+        return "ClientEndpoint{"
+                + "clientUuid=" + clientUuid
+                + ", clientName=" + clientName
+                + ", authenticated=" + authenticated
+                + ", clientVersion=" + clientVersion
```

Comment:
it looks to still to much leaking too much info according to the issue, right?

Reply:
These info comes from the client already, isn't it? I don't think it is sensitive information.

Reply:
I agree with Serkan. The only information related to the server-side here in this message is the `creationTime` and `authenticated` which do not leak security-related information in my view. Maybe we can remove those two as well just to be sure? They don't seem to be particularly useful anyway in the exception message propagated to the un-authenticated client

Reply:
Ok removing to be sure.